src/shared/mocks/route-components-props-mock.ts

```diff
@@ -0,0 +1,48 @@
+import { Href, UnregisterCallback } from 'history';
+import { RouteComponentProps } from 'react-router';
+
+/**
+ * This is to mock out the dependencies for react router
+ */
+export function getMockRouterProps<P>(data: P) {
+	const location = {
+		hash: '',
+		key: '',
+		pathname: '',
+		search: '',
+		state: {},
+	};
+
+	const props: RouteComponentProps<P> = {
+		match: {
+			isExact: true,
+			params: data,
+			path: '',
+			url: '',
+		},
+		location,
+		history: {
```

Comment:
Je kan hier `createBrowserHistory()` van de history package voor gebruiken in uw tests